### ISSUE TITLE
tweak resource ids for :outcome-panel/visit and :outcome-panel/for-frequent-updates

### ISSUE THREAD

**Issue by oakmac** [LuminareMed/quickscreen-translations#10]
Follow up from PR #9 

> The current mapping of the key names :outcome-panel/visit and :outcome-panel/for-frequent-updates is technically incorrect. The translation in Korean is written as: "[links] by visiting // get frequent updates // " instead of "visit [links] // for frequent updates" in English. If the link is put in between the two keys, the Korean text would not make much sense.

How can we re-write the text here in order to handle this situation? [tempura](https://github.com/ptaoussanis/tempura) supports some light variable interpolation. Maybe we can do something like:

```edn
:outcome-panel/url-for-updates "Please visit % for frequent updates"
```

Would that work @semper21 ?

**Comment by semper21** [comment]
Yeah that would totally work!

**Comment by oakmac** [comment]
@semper21 updated the dictionaries to use positional arguments, see https://github.com/LuminareMed/quickscreen-translations/commit/c4b6a041f009cd339394db8605baf8b2c9bc0724

for this issue specifically see https://github.com/LuminareMed/quickscreen-translations/blob/master/ko.edn#L95-L96

**Comment by oakmac** [comment]
Fixed with PR #13
